src/main/java/uk/gov/register/resources/RecordResource.java

```diff
@@ -20,18 +20,18 @@
 @Path("/")
 public class RecordResource {
     private final HttpServletResponseAdapter httpServletResponseAdapter;
+    private final RequestContext requestContext;
```

Comment:
`requestContext` is only used in the constructor, why are you adding it as property?